documentation/architecture.asciidoc

```diff
@@ -3,7 +3,7 @@ toc::[]
 :idprefix:
 :idseparator: -
 
-There are many different views on what is summarized by the term _architecture_. First we introduce the xref:key-principles[key principles] and xref:architecture-principles[architecture principles] of devonfw. Then we go into details of the the xref:application-architecture[architecture of an application].
+There are many different views that are summarized by the term _architecture_. First, we will introduce the xref:key-principles[key principles] and xref:architecture-principles[architecture principles] of devonfw. Then, we will go into details of the the xref:application-architecture[architecture of an application].
```

Comment:
IMHO the original idea was to say that _architecture_ is a common term but different people have different understandings and views on this. With the change it now IMHO says that _architecture_ summarizes different views (like the view of a developer, operator, business-analyst?) in one term.